cmd/qri.go

```diff
@@ -162,6 +155,11 @@ func (o *QriOptions) Instance() *lib.Instance {
 	return o.inst
 }
 
+// QriPath returns the path to the qri directory
```

Comment:
Would prefer some more specificity. This could mean the "qri directory" where the binary is installed, or where Desktop exists, or where I store my datasets. How about "QriPath returns the path where qri data and configuration exist"